geode-core/src/main/java/org/apache/geode/internal/cache/properties.html

```diff
@@ -383,7 +383,7 @@ <h1>Properties, in Alphabetical Order</h1>
 <dd>
 <em>Public:</em> false
 <p>
-<em>Integer</em> (default is 1)
+<em>Integer</em> (default is 200)
```

Comment:
not sure if I'm missing something, but it looks like default will be 200, if the machine has less than 6 cores, other wise the default is num processors * 32. Not sure how significant this info is to our users?

Reply:
Thanks for pointing this out, I'll update the description.